backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Services/CommitFilesService.cs

```diff
@@ -0,0 +1,207 @@
+            var item = staged.DbTableStructures?.FirstOrDefault(x => x.Name == selected);
+            if (item != null)
+            {
+                var blobContent = Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(item));
+                var file = new CommitFileDto
+                {
+                    BlobId = $"{item.Schema}-{item.Name}".ToLower(),
+                    FileName = item.Name,
+                    FileType = Shared.Enums.DatabaseItemType.Table
+                };
+                await SaveToBlob(file, selectedItems.CommitId, blobContent);
+                saved.Add(file);
+            }
+        }
+        foreach (var selected in selectedItems.Constraints)
+        {
+            var item = staged.DbConstraints.FirstOrDefault(x => x.Name == selected);
+            if (item != null)
+            {
+                var blobContent = Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(item));
+                var file = new CommitFileDto
+                {
+                    BlobId = $"{item.Schema}-{item.Name}".ToLower(),
+                    FileName = item.Name,
+                    FileType = Shared.Enums.DatabaseItemType.Constraint
+                };
+                await SaveToBlob(file, selectedItems.CommitId, blobContent);
+                saved.Add(file);
+            }
+        }
+        foreach (var selected in selectedItems.Views)
+        {
+            var item = staged.DbViewsDetails.Details.FirstOrDefault(x => x.Name == selected);
+            if (item != null)
+            {
+                var blobContent = Encoding.UTF8.GetBytes(JsonConvert.SerializeObject(item));
+                var file = new CommitFileDto
+                {
+                    BlobId = $"{item.Schema}-{item.Name}".ToLower(),
+                    FileName = item.Name,
+                    FileType = Shared.Enums.DatabaseItemType.View
+                };
+                await SaveToBlob(file, selectedItems.CommitId, blobContent);
+                saved.Add(file);
+            }
+        }
+        foreach (var selected in selectedItems.Types)
+        {
+            // TODO
+        }
+
+        return saved;
+    }
+
+    private async Task SaveToBlob(CommitFileDto dto, int commitId, byte[] content)
+    {
+        var blob = new Blob
+        {
+            Id = dto.BlobId,
+            ContentType = "application/json",
+            Content = content
+        };
+        await _blobService.UploadAsync($"{commitId}-{dto.FileType}".ToLower(), blob);
+    }
+
+    private async Task<DbStructureDto> GetStagedAsync(string changesGuid)
+    {
+        var blob = await _blobService.DownloadAsync(_containerName, changesGuid);
+
+        var json = Encoding.UTF8.GetString(blob.Content ?? throw new ArgumentNullException());
+        
+        return JsonConvert.DeserializeObject<DbStructureDto>(json) ?? throw new JsonException($"{nameof(DbStructureDto)} deserialized as null!");
+    }
+
+    public DbStructureDto GetTestStructure()
```

Comment:
`public DbStructureDto GetTestStructure()`

do we still need it (now we can connect to real DBs)?

Reply:
Removed